latest/docs-ref-conceptual/install-azure-cli.md

```diff
@@ -101,15 +101,63 @@ For Debian/Ubuntu based systems, you can install Azure CLI 2.0 via `apt-get`.
 
 3.  Run the CLI from the command prompt with the `az` command.
 
+## Install on RHEL, Fedora, and CentOS with yum
+
+For any distribution which is based off of RedHat and contains the `yum` package manager, you can install Azure CLI 2.0 via `yum`.
+
+1. Import the Microsoft repository key:
+
+   ```bash
+   sudo rpm --import https://packages.microsoft.com/keys/microsoft.asc
+   ```
+
+2. Create local `azure-cli` repository information:
+
+   ```bash
+   sudo sh -c 'echo -e "[azure-cli]\nname=Azure CLI\nbaseurl=https://packages.microsoft.com/yumrepos/azure-cli\nenabled=1\ngpgcheck=1\ngpgkey=https://packages.microsoft.com/keys/microsoft.asc" > /etc/yum.repos.d/azure-cli.repo'
+   ```
+
+3. Update the `yum` package index and install:
+
+   ```bash
+   yum check-update
+   sudo yum install azure-cli
+   ```
+
+4. Run the CLI from the command prompt with the `az` command.
+
+## Install on openSUSE and SLE with zypper
+
+1. Import the Microsoft repository key:
+
+   ```bash
+   sudo rpm --import https://packages.microsoft.com/keys/microsoft.asc
+   ```
+
+2. Create local `azure-cli` repository information:
+
+   ```bash
+   sudo sh -c 'echo -e "[azure-cli]\nname=Azure CLI\nbaseurl=https://packages.microsoft.com/yumrepos/azure-cli\nenabled=1\ntype=rpm-md\ngpgcheck=1\ngpgkey=https://packages.microsoft.com/keys/microsoft.asc" > /etc/zypp/repos.d/azure-cli.repo'
+   ```
+
+3. Update the `zypper` package index and install:
+
+   ```bash
+   sudo zypper refresh
+   sudo zypper install code
```

Comment:
This installs VSCode. Should be azure-cli

Reply:
I thought something about this looked weird... fixed.